src/styles/index.scss

```diff
@@ -15,3 +15,9 @@
 .usa-search [type='search']::-webkit-search-cancel-button {
   display: none;
 }
+
+// This just allows buttons as well as links in the LanguageSelector
+// It can be removed when https://github.com/uswds/uswds/issues/5409 is fixed
+.usa-language__submenu .usa-language__submenu-item button {
+  @extend a;
+}
```

Comment:
I'm wondering if the failing lint check is related to this?

Reply:
it is, I [commented above](https://github.com/trussworks/react-uswds/pull/2531#issuecomment-1659272663)

Reply:
Sorry I missed that! Yeah that is a weird one. It may need to be overridden like you said, but not sure.

Reply:
@shkeating any ideas?